plugins/catalog/src/routes.ts

```diff
@@ -36,6 +36,11 @@ export const createFromTemplateRouteRef = createExternalRouteRef({
   params: ['namespace', 'templateName'],
 });
 
+export const unregisterRedirectRouteRef = createExternalRouteRef({
+  id: 'unregister-redirect',
```

Comment:
we normally try namespace these. something like `catalog:unregister-redirect` or something works.